string intent
float64 confidence
rasa_ros/RasaOrderMsg[] entities